src/ui/public/field_editor/field_editor.js

```diff
@@ -87,6 +87,10 @@ uiModules
         });
       };
 
+      self.isSupportedLang = function (lang) {
+        return _.contains(['expression', 'painless'], lang);
```

Comment:
Extracting the array of supported languages into a constant somewhere would allow to reduce duplication with the scripted field list.